cloudbuild.yaml

```diff
@@ -9,18 +9,31 @@ imports:
     name: pubsub.jinja
   - path: deployment/server.jinja
     name: server.jinja
+  - path: deployment/secret.jinja
+    name: secret.jinja
+  - path: deployment/iam.jinja
+    name: iam.jinja
 
 resources:
+  - name: service-account
+    type: iam.jinja
+
   - name: pubsub
     type: pubsub.jinja
+    metadata:
+      dependsOn:
+      - service-account
+
   - name: mu-table
     type: bigquery.jinja
+
   - name: mu-build
     type: docker.jinja
     properties:
       project: server
       version: 0.1.0
-      branch: master
+      branch: kubernetesSecret
```

Comment:
I have to change it before merge. Right now, I am keeping the name of the current branch so you can try all the deployment from this branch